main.tf

```diff
@@ -9,6 +9,12 @@ resource "aws_instance" "webserver" {
   tags = {
     Name = "webserver"
   }
+
+  metadata_options {
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure detailed monitoring is enabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances without a detailed monitoring could have an impact on incident response.
 
**Remediation**
It is recommended to enable detailed monitoring for EC2 instances. This could be achieved by changing monitoring attribute in IAC to 'true' or launch every new EC2 instance with monitoring check box ticked.

source: accurics-stageone